app/src/app/components/specmath-modal/modal.component.ts

```diff
@@ -26,30 +26,33 @@ enum steps {
 
 type StepOptions = {
   [key in steps]: {
```

Comment:
nit: I couldn't leave the comment higher but take a look at the naming conventions for enums: https://www.typescriptlang.org/docs/handbook/enums.html. Maybe 'Step' instead of 'steps'?

Reply:
Updated